src/components/side_panel/carousel_panel/carousel_panel.ts

```diff
@@ -118,6 +118,15 @@ export class CarouselPanel extends Component<Props, SpreadsheetChildEnv> {
     });
   }
 
+  popOutCarouselItem(item: CarouselItem, ev: MouseEvent) {
+    if (item.type !== "chart") return;
+    this.env.model.dispatch("POPOUT_CHART_FROM_CAROUSEL", {
+      sheetId: this.env.model.getters.getActiveSheetId(),
```

Comment:
All calls to activeSheetId here are zrong as we can change the active sheet while the sidepanel is open.

We could either
- close the side panel if we change of activeSheet (update the `computeState` in the sidePanelRegistry)
- fetch the sheetId of the carousel and store it locally in the component